coderd/files/cache.go

```diff
@@ -140,20 +141,36 @@ type cacheEntry struct {
 
 type fetcher func(context.Context, uuid.UUID) (CacheEntryValue, error)
 
+var (
+	_ fs.FS     = (*CloseFS)(nil)
+	_ io.Closer = (*CloseFS)(nil)
```

Comment:
I don't like having `Close` return an error just so that it can implement `Closer`. we don't really get any benefit out of it matching this interface.